.github/workflows/ci.yaml

```diff
@@ -163,3 +163,12 @@ jobs:
           file: ${{ matrix.connector }}/Dockerfile
           push: true
           tags: ghcr.io/estuary/${{ matrix.connector }}:${{ steps.prep.outputs.tag }}
```

Comment:
does tags take an array ? can we add it here?

Reply:
It does take an array, but the semantics of the two push actions are different. The first one will push a hash-tagged image whenever the CI runs, including on not-yet-merged PRs, while the second one needs to only push the `dev`-tagged image when it's been committed.

I considered trying to make it an array which may or may not include the `dev` tag, but figuring out how to do that conditionally in Github Actions YAML was beyond me.

Reply:
Ah my bad, I failed to note the `if` conditional.